### PR TITLE
upgrade mapbox v2 and destory resize event

### PR DESCRIPTION
**Pull request by purpleplum**
Resolves # .

# Why is this change necessary?
Fix issue 111 and 113

# How does it address the issue?
Upgrade mapbox to v2 and modify resize event listener in clickcatcher.

# What side effects does it have?
None

### CODE REVIEW

File: .yarnrc.yml
Comment (kaden-weber) [inline .yarnrc.yml:1]: This file should not be added at the top level, add changes to the `ember/.yarnrc.yml` instead.

**Comment by kaden-weber** [comment]
These changes are causing an issue with the popups when mousing over a development: 

Note the popup stuck at the origin in the top left:
Screenshot 2023-10-17 at 4 06 50 PM

Expected behavior happens sometimes: 
Screenshot 2023-10-17 at 4 09 33 PM

Is there a use case for updating mapbox? I don't think we will want to do the upgrade without avoiding this issue.

File: .yarnrc.yml
Comment (purpleplum) [reply .yarnrc.yml:1]: I must have saved it in the wrong place.

**Comment by purpleplum** [comment]
> These changes are causing an issue with the popups when mousing over a development:
> 
> Note the popup stuck at the origin in the top left: Screenshot 2023-10-17 at 4 06 50 PM
> 
> Expected behavior happens sometimes: Screenshot 2023-10-17 at 4 09 33 PM
> 
> Is there a use case for updating mapbox? I don't think we will want to do the upgrade without avoiding this issue.

Interesting, I am not seeing the same popup issue on my end.  Might be a browser problem? Mapbox is already version 3. We don't want to be too much behind. In addition, it seems to be an easy thing to upgrade.

**Comment by purpleplum** [comment]
The popup issue might be related to issue 68 as well.

**Pull request closed without merging**